### DISCUSSION TITLE
Reanimated zoom issue

### DISCUSSION THREAD

**Discussion by AmitGunjan** [software-mansion/react-native-reanimated#5527] [Q&A]
On zoom view is getting hide behind the below view, is there is any way to overap with top and bottome view after zoom?

**Comment by MatiPl01** [comment]
Hi!

I don't understand what problem you are trying to solve. Could you provide a reproducible code example if you still need help with this issue?

Your issue seems to be a problem with the zoomed view `zIndex`. You should be able to solve it by setting higher `zIndex` of the zoomed component.
